alpha/scripts/utils/updateTagAccordingToRefenceId.php

```diff
@@ -0,0 +1,41 @@
+<?php
+require_once('/opt/kaltura/app/alpha/scripts/bootstrap.php');
+
+if ($argc < 3)
+{
+	die("Usage: php $argv[0] entryIdsFile referenceIdsTagFile.csv <realrun | dryrun> \n");
+}
+
+$entryIdsFile = $argv[1] ;
+$referenceIdsTagFile = $argv[2];
+$dryrun = true;
+if($argc == 4 && $argv[3] == 'realrun')
+	$dryrun = false;
+
+$entryIds = file ($entryIdsFile) or die ('Could not read file'."\n");
+$referenceIdsTagList = file ($referenceIdsTagFile) or die ('Could not read file'."\n");
+
+foreach ($entryIds as $entryId)
+{
+	$entryId = trim($entryId);
+	$entry = entryPeer::retrieveByPK($entryId);
+	if ($entry)
+	{
+		$entryReferenceId = $entry->getReferenceID();
+		foreach ($referenceIdsTagList as $referenceIdsTag)
+		{
+			$referenceIdsTag = trim($referenceIdsTag);
+			list($referenceId, $tag) = explode(',', $referenceIdsTag);
+			if ($referenceId && $tag && $entryReferenceId === $referenceId)
+			{
+				$entryTags = $entry->getTags();
+				$entry->setTags($entryTags . ',__' . $tag);

```

Comment:
@ravitshalem  
1. We want to add the new tag to the existing tags so just making sure, setTags doesn't override existing tags, right? it just adding the new one?
2. please use these lines to convert the Content Group value into tag: (to be consistent with how we handle other "hidden" tags in KMS)
$pattern = array('/^/', '/[^[:alnum:]]/u');
$replcement = array('__', '_');
$tag = preg_replace($pattern, $replcement, $tag);